source/python_list/purpose_of___version___in_std_library.rst

```diff
@@ -0,0 +1,58 @@
+###############################################################
```

Comment:
This is a pretty minor issue -- if you want to put in pages like this, we may want to create a bit more structure.

I see this project as mostly about capturing:

A) common issues that will come up again
B) collecting / summarizing large threads that are hard to find and read on the list archives.

Reply:
@PythonCHB ah yes in priority, then earmark them then we come to those like this.

Reply:
or use issues for that...